mbed_build/_internal/cmake_renderer.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2020 Arm Mbed. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+import pathlib
+
+import jinja2
+
+TEMPLATES_DIRECTORY = pathlib.Path("_internal", "templates")
+TEMPLATE_NAME = "CMakeLists.tmpl"
+
+
+def _render_cmakelists_template() -> str:
```

Comment:
Would it be possible to put the public functions before the private functions? I generally think it's easier to read that way.

Reply:
Oh yes! Sorry I keep forgetting to do that!

Reply:
41bc7d7